Makefile

```diff
@@ -19,3 +19,9 @@ endif
 .PHONY: tidy
 tidy: format lint
 
+# Removes the directory that contains bytecode cache files
+# that are automatically generated by python.
+.PHONY: clean
+clean:
+	find . -type f -name "*.pyc" | xargs rm -fr
+	find . -type d -name __pycache__ | xargs rm -fr
```

Comment:
This is convenience function for developers. Run `make clean` to delete pythons cache files.